fog/sample-paykit/src/client.rs

```diff
@@ -173,6 +176,20 @@ impl Client {
         self.tx_data.get_latest_block_version()
     }
 
+    /// Get the currently cached fee-map
+    ///
+    /// If the cache is empty, then we get fresh data from consensus.
+    pub fn get_fee_map(&mut self) -> Result<FeeMap> {
```

Comment:
I think it's useful in the test-client, I want to use it in the test client to avoid making the operator pass the fee as a parameter. We can make a ticket for that if you like

Reply:
I was looking at how the test client actually gets this data, it looks that we already have a function for this that I missed. Going to refactor